src/MembershipsSDKAdapter.js

```diff
@@ -89,7 +89,7 @@ function getMembers(sdkMembers) {
   members = sortMeetingMembers(members);
 
   return members.map((member) => ({
-    ID: member.id,
+    ID: member.participant && member.participant.person && member.participant.person.id,
```

Comment:
as we need the `person` 2 times now, let's put it in a var:
```js
const person = (member.participant && member.participant.person) || {};
return {
  ID: person.id,
  orgID: person.orgId,
  ...
};
```

on the other hand, this means introducing the "return" statement which makes the syntax a bit more cluttered. This is one place where the `?.` operator would help:

```js
ID: member.participant?.person?.id,
orgID: member.participant?.person?.orgId,
...
```

@lalli-flores If it's very unlikely that an SDK `member` wouldn't have a defined `participant` property, we can drop the multiple tests and just go with `ID: member.participant.person.id`.

Reply:
I'm not 100% sure but they be not be the same thing, for instance, if a user joins using a Webex device, then it might be a user but not a person (as it is a device). I would need to confirm my assumption.

This is still not mergeable, I left a comment yesterday that hasn't been addressed, but we need to make sure we return a Hydra ID and not a UUID